packages/ci/build-prod-images.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 echo "Building backend docker image..."
-docker build -t litefarm/backend:latest -f ./packages/api/prod.Dockerfile ./packages/
+docker build --platform linux/amd64 -t litefarm/backend:latest -f ./packages/api/prod.Dockerfile ./packages/
```

Comment:
* Let's add the `--platform` option to both `docker build [...]` calls
* I suggest to use long version of parameters, `--tag` instead of `-t`, `--file` instead of `-f`
  * The rationale behind using long parameter names is to increase the readability of the script, specially for those not familiar with the concepts behind the program, e.g. What's a _tag_ when building a Docker image?